recipes/flake8/meta.yaml

```diff
@@ -1,63 +0,0 @@
-{%set name = "flake8" %}
```

Comment:
@jakirkham : if I undelete that file, there will be two meta.yaml files. Actually, while building flake8, I forgot to create a new branch for flake8, and created recipe at master of the forked repo. When I made a branch for django-configurations, it had a directory flake8 with an yaml file. So, I deleted that file.
If I undelete that file, there will be two yaml files. Will that cause a problem?